CMakeLists.txt

```diff
@@ -87,4 +87,5 @@ if(REALM_JS_BUILD_CORE_FROM_SOURCE)
     add_subdirectory(vendor/realm-core EXCLUDE_FROM_ALL)
 endif()
 
-add_subdirectory(src)
\ No newline at end of file
+add_subdirectory(src)
+add_definitions(-DNAPI_VERSION=3)
```

Comment:
Could this be moved somewhere more Node.js specific?

Reply:
`dependencies.list` has `NAPI_VERSION=4`. See also https://github.com/realm/realm-js/blob/master/src/node/node_value.hpp#L77.

When node v12.x has end-of-life (April 2022), we will bump to NAPI version 5.